src/reference-data-entities/category.ts

```diff
@@ -0,0 +1,7 @@
+import { Entity, BaseEntity, PrimaryColumn } from 'typeorm';
+
+@Entity({ name: 'categories' })
```

Comment:
Why plural here?

Reply:
Its what was in the original SQL and I went with it.  Figured this was Owains area of expertise and I wouldn't meddle to much.